moveit_core/utils/CMakeLists.txt

```diff
@@ -19,7 +24,7 @@ target_include_directories(moveit_test_utils PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include/moveit_core>
 )
-target_link_libraries(moveit_test_utils moveit_robot_model moveit_kinematics_base)
+target_link_libraries(moveit_test_utils moveit_robot_model moveit_kinematics_base rsl::rsl)
```

Comment:
```suggestion
```
I don't see this target using `rsl::rsl` in this branch so I think this can be removed.